driver/optmain.ml

```diff
@@ -239,6 +239,9 @@ let main () =
   try
     readenv ppf Before_args;
     Clflags.add_arguments __LOC__ (Arch.command_line_options @ Options.list);
+    Clflags.add_arguments __LOC__
+      ["-depend", Arg.Unit Makedepend.main_from_option,
+       "ARGS Compute dependencies between source files"];
```

Comment:
You should refer the user to `ocamlc -depend -help` for an explanation of ARGS, or even just replace this message with `"<options> run ocamldep on <options>"`.